README.md

```diff
@@ -78,14 +78,23 @@ New minor versions may add additional features to the API.
 ### Installing
 
 To start using Bolt, install Go and run `go get`:
-
 ```sh
-$ go get go.etcd.io/bbolt/...
+$ go get go.etcd.io/bbolt@latest
 ```
 
-This will retrieve the library and install the `bolt` command line utility into
-your `$GOBIN` path.
+This will retrieve the library and update your go.mod and go.sum files. 
+
+To run the command line utility, execute:
```

Comment:
[sorry] I thought 'edit' will add this as a suggestion, not as a commit on top. But please take a look.

Reply:
Looks good to me. Please signoff the commit.

Reply:
I'm afraid I cannot force push to your branch to fix it... I only can propose a new commit.

Please squash it or remove my commit. 

I'm sorry for the trouble.

Reply:
NO problem. Let me take care of it.

Reply:
resolved.